corehq/ex-submodules/pillowtop/feed/interface.py

```diff
@@ -60,9 +61,10 @@ def get_document(self):
         if not self.document and self.document_store and not self._document_checked:
             try:
                 self.document = self.document_store.get_document(self.id)
-            except DocumentNotFoundError:
+            except DocumentNotFoundError, e:
```

Comment:
if you use flake8 to check code you can install https://docs.openstack.org/developer/hacking/#python-3-x-compatibility. i attempted to add it to dimagimon but never quite figured it out